MANIFEST.in

```diff
@@ -0,0 +1 @@
+include requirements.txt requirements_tests.txt README.md
```

Comment:
Is it important to include the requirements files in the sdist?

I thought README.md should get picked up automatically, but I was wrong! It only scoops up README, README.txt, or README.rst by default; https://docs.python.org/3.8/distutils/sourcedist.html#specifying-the-files-to-distribute

Reply:
Oh: yes, it's very important if `setup.py` reads them.

Reply:
Yeah, it was only required so that `setup.py` can read it.